fix(*): Update Helm to specify v2 parts

hickeyma opened the pull request:
Adds clarification to better distinguish specific v2 parts. Also, some small typo fixes.

hickeyma committed:
Update Helm v2

Adding clarification to the Helm v2 tutorial.

spzala commented on README.md:
```diff
@@ -1,5 +1,22 @@
-# Introduction to Helm 
+# Introduction to Helm v2
```
nit and you can decide but I think no need to add v3 in the title here...the name of repo shows that it's about v2. In the introduction there is nothing specific about v2 so I would leave the title of this Intro section as is.

hickeyma replied:
Agree, removed

spzala commented on tutorial/Lab4/README.md:
```diff
@@ -6,7 +6,7 @@ A means of sharing is a chart repository, which is a location where packaged cha
 # Using charts from a public repository
 
-Helm charts can be available on a remote repository or in a local environment/repository. The remote repositories can be public like [Helm Charts](https://github.com/helm/charts) or [IBM Helm Charts](https://github.com/IBM/charts), or hosted repositories like on Google Cloud Storage or GitHub. Refer to [Helm Chart Repository Guide](https://v2.helm.sh/docs/developing_charts/#the-chart-repository-g ...
+Helm charts can be available on a remote repository or in a local environment/repository. The remote repositories can be public like [Bitnami Charts](https://github.com/bitnami/charts) or [IBM Helm Charts](https://github.com/IBM/charts), or hosted repositories like on Google Cloud Storage or GitHub. Refer to [Helm Chart Repository Guide](https://helm.sh/docs/topics/chart_repository/) for more det ...
```
Sorry if you have answered it earlier as part of my similar comment on #31 but I don't recall, why we are replacing the Helm Charts link with Bitnami? Thanks!

hickeyma replied:
Not a bother. This is because the Helm chart repository is going out of support in parallel with Helm v2 support. Charts will need to be hosted by the chart maintainer. Details on this: https://github.com/helm/charts#status-of-the-project

spzala reviewed (commented):
@hickeyma thanks and lgtm.. couple of small comments though.

hickeyma committed:
Update after review

Review comments:
- https://github.com/IBM/helm101/pull/32#pullrequestreview-372263572

hickeyma commented:
Thanks @spzala for review and comments. Updated and ready for review again.

spzala commented:
@hickeyma thanks for addressing the comments. Merging it.
